src/vend-number.js

```diff
@@ -27,153 +29,166 @@ const VendNumber = {
      * deal with.
      * https://github.com/MikeMcl/bignumber.js/issues/11
      */
-    return BigNumber(value ? value.toString() : 0)
-  },
+    super(value ? value.toString() : 0)
+  }
+}
 
-  /**
-   * Rounds a value to a specified number of decimal points.
-   *
-   * @method round
-   * @param value {Number|String}
-   *      The value to round to the provided number decimal points or two.
-   *
-   * @param [decimalPoints=2]
-   *      The number of decimal points to round the passed value to, or two.
-   *
-   * @return {String} The rounded value.
-   */
-  round: function (value, decimalPoints) {
-    // Convert to VendNumber if not already.
-    value = (value instanceof BigNumber) ? value : new VendNumber.VendNumber(value)
+/**
+ * Quick convenience function for creating VendNumber instances. E.g. `vn(123)` is the same as `new VendNumber(123)`.
+ *
+ * @method vn
+ * @static
+ *
+ * @param value {Number | String}
+ *        The value to make a VendNumber
+ *
+ * @return {VendNumber} A VendNumber instance for the given value
+ */
+VendNumber.vn = function (value) {
+  return new VendNumber(value)
+}
 
-    // 2dp by default.
-    decimalPoints = (typeof decimalPoints === 'number') ? decimalPoints : 2
+/**
+ * Rounds a value to a specified number of decimal points.
+ *
+ * @method round
+ * @static
+ *
+ * @param value {Number|String}
+ *        The value to round to the provided number decimal points or two.
+ *
+ * @param [decimalPoints=2]
+ *        The number of decimal points to round the passed value to, or two.
+ *
+ * @return {String} The rounded value.
+ */
+VendNumber.round = function (value, decimalPoints) {
```

Comment:
Not crazy about using arrow functions in this situation as then the value of `this` inside those functions will be the global variable rather than the `VendNumber` class. I know we're not using `this` in these functions so technically it doesn't matter but still doesn't feel right to me.